app/views/pages/best-practices/qa/installing-testcafe.liquid

```diff
@@ -47,6 +55,13 @@ If TestCafe has been installed correctly, running this command displays the vers
 
 After installed, you can run TestCafe from a command line by using the `testcafe` command. See a list of available commands and options [here](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html).
 
+## Examples
+
+See complete TestCafe test suites and file organization examples on platformOS sites in the following public repos:
+
+* [https://github.com/mdyd-dev/nearme-documentation/tree/master/tests/e2e](https://github.com/mdyd-dev/nearme-documentation/tree/master/tests/e2e)
+* [https://github.com/mdyd-dev/marketplace-nearme-example/tree/master/tests/e2e](https://github.com/mdyd-dev/marketplace-nearme-example/tree/master/tests/e2e)
```

Comment:
Actually, examples are split into modules and tests are embedded inside module directory.